classes/class-woothemes-sensei-certificate-templates.php

```diff
@@ -889,10 +889,17 @@ private function get_item_meta_value( $meta_name ) {
 	 * @return void
 	 */
 	public function save_post_meta( $post_key = '', $post_id = 0 ) {
+		if (
+			empty( $_POST['course_certificates_meta_nonce'] )
+			// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- Leave nonce value unmodified.
+			|| ! wp_verify_nonce( wp_unslash( $_POST['course_certificates_meta_nonce'] ), 'course_certificates_save_data' )
+		) {
+			return;
+		}
 
 		// Get the meta key.
 		$meta_key       = '_' . $post_key;
-		$new_meta_value = ( isset( $_POST[ $post_key ] ) ? sanitize_html_class( $_POST[ $post_key ] ) : '' );
+		$new_meta_value = isset( $_POST[ $post_key ] ) ? intval( $_POST[ $post_key ] ) : '';
```

Comment:
Just wanted to leave a note for myself: this does indeed seem to only be intval.